.github/workflows/release.yaml

```diff
@@ -0,0 +1,41 @@
+name: Build and relase image
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '1.14.8' # The Go version to download (if necessary) and use.
+    - run: scripts/version
+    - run: |
+        export LINKFLAGS="-extldflags -static -s"
```

Comment:
what are the semantics of additional link flags

Reply:
removes symbol table and compilies statically